.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -0,0 +1,106 @@
+jobs:
+# Configure, build, install, and test job
+- job: 'linux_build'
+  displayName: 'Linux Builds'
+  pool:
+    vmImage: 'ubuntu-latest'
+  timeoutInMinutes: 120
+  variables:
+    CTEST_OUTPUT_ON_FAILURE: 1
+  strategy:
+    maxParallel: 8
+    matrix:
+      gcc_build:
+        CXX_COMPILER: 'g++'
+        PYTHON_VER: '3.7'
+        C_COMPILER: 'gcc'
+        Fortran_COMPILER: 'gfortran'
+        BUILD_TYPE: 'release'
+
+  steps:
+  - script: |
+      uname -a
+      free -m
+      df -h
+      ulimit -a
+      mkdir -p $HOME/miniconda
+      mkdir -p $HOME/build
+
+      ./.scripts/miniconda.sh
+    displayName: 'Build Miniconda'
+
+  - script: |
+      export PATH="$HOME/miniconda/bin:$PATH"
+      conda create -q -n p4env python=$PYTHON_VER psi4 --only-deps -c psi4/label/dev
+      conda activate p4env
+      conda install dftd3 ^
+                    gcp ^
+                    resp ^
+                    pycppe ^
+                    pylibefp ^
+                    snsmp2 ^
+                    mp2d ^
+                    pytest-xdist ^
+                    -c psi4/label/dev
+       conda install qcelemental ^
+                     qcengine ^
+                     pybind11 ^
+                     -c conda-forge
```

Comment:
why c-f not psi4/label/dev?

Reply:
I pulled this from Travis.

Reply:
figure out why it's from c-f. this makes sure psi is using latest, even if I'm behind on psi4-dev packages.